include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -820,7 +842,9 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", []> {
 
   let arguments = (
     ins AnyMemRef:$buffer,
-        OptionalAttr<AIEI32Attr>:$offset,
+        // in words (multiples of 32b) not bytes
+        DefaultValuedOptionalAttr<AIEI32Attr, "0">:$offset,
```

Comment:
This doesn't make sense to me...shouldn't it me in multiples of the element type?

Reply:
comment is wrong but impl is right (ie what you said)

Reply:
But actually I realize I lied and didn't patch the n-d addressing too.

Reply:
Top commit patches size and stride as well; libxaie says 

```
* @note		None. The stepsize and wrap parameters operate at 32 bit
*		granularity. 
```

but I take that to mean just stride (what they're calling stepsize) right? Because what would it even mean for size ("wrap") to be at the 32b granularity (it's the loop bound...). Sanity check here would be much appreciated @stephenneuendorffer .